tox.ini

```diff
@@ -67,7 +67,7 @@ commands =
 # see: http://flake8.readthedocs.io/en/latest/config.html#settings
 show-source = True
 ignore = E123,E125
-builtins = _
+builtins = NS
```

Comment:
We should have both `_` and `NS` under builtins

Reply:
ah my fault, i will change it

Reply:
no need _ ? @r0h4n